common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -117,6 +117,7 @@ def B(*args, **kwargs):
 AUTH_ENTRY_DASHBOARD = 'dashboard'
 AUTH_ENTRY_LOGIN = 'login'
 AUTH_ENTRY_REGISTER = 'register'
+AUTH_ENTRY_ACCOUNT_SETTINGS = 'account_settings'
```

Comment:
Are we no longer supporting `AUTH_ENTRY_DASHBOARD`?  If so, we should remove it.